graders/java/JUnitAutograder.java

```diff
@@ -98,9 +99,9 @@ public void runTests() throws UngradableException {
 
         for (String classSrcName : testClasses) {
             String className = classSrcName
-                .replaceFirst("^/grade/tests/junit/", "")
-                .replaceFirst("\\.java$", "")
-                .replaceAll("/", ".");
+                    .replaceFirst("^/grade/tests/junit/", "")
+                    .replaceFirst("\\.java$", "")
+                    .replaceAll("/", ".");
```

Comment:
Maybe this is just my lack of familiarity with Java conventions, but why is this indented 8 spaces when everything is indented 4?

Reply:
This is usually the case when it's the continuation of a long line/statement instead of a block indentation.